### PR TITLE
Add definitions to Doxygen comments

### PR DESCRIPTION
**Pull request by bmaclach**
This PR contributes to #1783 by adding definitions to the Doxygen documentation in the generated code. For now, we show all definitions. 

At first when I enabled definitions, most of GlassBR's definitions were not showing. I realized it was because there were many quantities in GlassBR that were defined completely independently from their corresponding concepts. In other words, the same "thing" was being defined twice from scratch, once as a concept and once as a quantity, with different UIDs. I attempted to fix this by tying together quantities with their definitions, but it's possible I made mistakes and matched a quantity with an inappropriate definition. Also, in many cases the term for the concept was slightly different from the term for the quantity. The term for the concept is now the one that is used everywhere, which is why there were some changes to the stable SRS. So @smiths when you review stable changes be on the lookout for bad definitions or insufficient terms. If I made mistakes or you think any of the terms/definitions should be changed, let me know.

### CODE REVIEW

File: code/stable/glassbr/SRS/GlassBR_SRS.tex [hunk 1]
```diff
@@ -75,15 +75,15 @@ \subsection{Table of Symbols}
 \\
 $b$ & Plate width (short dimension) & m
 \\
-$capacity$ & Capacity or load resistance & Pa
+$capacity$ & Capacity & Pa
```
Comment (smiths) [inline code/stable/glassbr/SRS/GlassBR_SRS.tex]: I prefer the old definition of capacity.  The new one does not say anything that the variable name itself does not already convey.

File: code/stable/glassbr/SRS/GlassBR_SRS.tex [hunk 2]
```diff
@@ -109,7 +109,7 @@ \subsection{Table of Symbols}
 \\
 $LDF$ & Load duration factor & --
 \\
-$Load$ & Applied load (demand) or pressure & Pa
+$Load$ & Load & Pa
```
Comment (smiths) [inline code/stable/glassbr/SRS/GlassBR_SRS.tex]: As for capacity, I prefer the old definition for Load.

File: code/stable/glassbr/SRS/GlassBR_SRS.tex [hunk 3]
```diff
@@ -127,7 +127,7 @@ \subsection{Table of Symbols}
 \\
 ${P_{\text{f}\text{tol}}}$ & Tolerable probability of failure & --
 \\
-$q$ & Applied load (demand) & Pa
+$q$ & Demand & Pa
```
Comment (smiths) [inline code/stable/glassbr/SRS/GlassBR_SRS.tex]: I prefer the old definition for q.

**Comment by bmaclach** [comment]
@smiths I made the requested changes in d9e1f1da5bcdd8443e92a06d0c9d22c44471454d. There was a small ripple effect to other parts of the SRS, so you probably want to review the new stable changes and confirm that this is what we want.

**Pull request merged**